source/crud/tests/unified/create-null-ids.yml

```diff
@@ -0,0 +1,288 @@
+description: CRUD ID Type Tests
+schemaVersion: '1.0'
+createEntities:
+  - client:
+      id: client
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: database
+      client: client
+      databaseName: crud_id
+  - collection:
+      id: collection
+      database: database
+      collectionName: type_tests
+tests:
+  - description: inserting _id with type undefined via insertOne
+    operations:
+      - name: dropCollection
```

Comment:
Why is an explicit `dropCollection` operation necessary in each test? Wouldn't `initialData` with an empty collection suffice?

If we keep this, it could be refactored as a YAML anchor.

Reply:
Migrated to initialData, that's better practice than including drop in the test.